kong/ids.go

```diff
@@ -103,6 +103,30 @@ func (cg *ConsumerGroup) FillID() error {
 	return nil
 }
 
+// FillID fills the ID of an entity. It is a no-op if the entity already has an ID.
+// ID is generated in a deterministic way using UUIDv5. The UUIDv5 namespace is different for each entity type.
```

Comment:
> The UUIDv5 namespace is different for each entity type.

What meaning does this have in here? I believe we can drop this sentence from the godoc.

Reply:
This is just copied from `FillID` method of other entities. I think this sentence here is used to tell that different entities with the same (entity wide unique) key will have different IDs because we use UUIDv5 and assign different namespaces for each kind of entities.

Reply:
Got it. But in the context of this method (which we're discussing) this sentence doesn't really make much sense does it?

This method is only relevant for `Vault`s, so other types (mentioned in the godoc here) do not make sense IMHO.

But as I look at it now, I believe the concealed shared bit of this method (shared map of generators) might have this effect 🤔 

Perhaps using the following wording would make more sense?

```suggestion
// ID is generated in a deterministic way using UUIDv5. The UUIDv5 namespace being used for generation is separate from other namespaces used for generating IDs for other types.
```

Reply:
Done.